internal/cmd/network/command_describe.go

```diff
@@ -0,0 +1,89 @@
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+
+	networking "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+)
+
+type out struct {
+	Id                       string                                          `human:"Id" serialized:"id"`
+	EnvironmentId            string                                          `human:"Environment" serialized:"environment_id"`
+	DisplayName              string                                          `human:"Display Name" serialized:"display_name"`
+	Cloud                    string                                          `human:"Cloud" serialized:"cloud"`
+	Region                   string                                          `human:"Region" serialized:"region"`
+	ConnectionTypes          networking.NetworkingV1ConnectionTypes          `human:"ConnectionTypes" serialized:"connection_types"`
+	Cidr                     string                                          `human:"Cidr" serialized:"cidr"`
+	Zones                    []string                                        `human:"Zones" serialized:"zone"`
+	DnsResolution            string                                          `human:"DnsResolution" serialized:"dns_resolution"`
+	Phase                    string                                          `human:"Phase" serialized:"phase"`
+	SupportedConnectionTypes networking.NetworkingV1SupportedConnectionTypes `human:"SupportedConnectionTypes" serialized:"supported_connection_types"`
+	ActiveConnectionTypes    networking.NetworkingV1ConnectionTypes          `human:"ActiveConnectionTypes" serialized:"active_connection_types"`
+	ReservedCidr             string                                          `human:"ReservedCidr" serialized:"reserved_cidr"`
+	ResourceUrl              string                                          `human:"Resource URL" serialized:"resource_url"`
+	ResourceName             string                                          `human:"Resource Name" serialized:"resource_name"`
+	CreatedAt                string                                          `human:"Created At" serialized:"created_at"`
+	UpdatedAt                string                                          `human:"Updated At" serialized:"updated_at"`
+	DeletedAt                string                                          `human:"Deleted At" serialized:"deleted_at"`
+}
+
+func (c *command) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe <id>",
+		Short: "Desribe a network.",
+		Args:  cobra.ExactArgs(1),
+		// ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		RunE: c.describe,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Describe Confluent Network "n-abcde1".`,
+				Code: `confluent network describe n-abcde1`,
+			},
+		),
+	}
+	pcmd.AddOutputFlag(cmd)
+	pcmd.AddClusterFlag(cmd, c.AuthenticatedCLICommand)
```

Comment:
our APIs won't take `cluster` as an argument to describe a network. check if the arg is needed

Reply:
nice catch, copy pasta these few lines and forgot to remove this arg